docs/source/conf.py

```diff
@@ -223,7 +223,7 @@ def is_stable(extra):
 htmlhelp_basename = 'ipythondoc'
 
 intersphinx_mapping = {'python': ('https://docs.python.org/3/', None),
-                       'rpy2': ('https://rpy2.readthedocs.io/en/version_2.8.x/', None),
+                       'rpy2': ('https://rpy2.readthedocs.io/en/latest/', None),
```

Comment:
You should explain in the commit message why you have updated this line, as it is not obvious why it relates to adding an Arm64 Travis-ci job.

Reply:
That has been fixed in master; I'll rebase so that it does not appear in the PR